slack-base/src/main/java/com/hubspot/slack/client/models/response/conversations/ConversationListResponseIF.java

```diff
@@ -0,0 +1,24 @@
+package com.hubspot.slack.client.models.response.conversations;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.conversations.Conversation;
+import com.hubspot.slack.client.models.response.ResponseMetadata;
+import com.hubspot.slack.client.models.response.SlackResponse;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ConversationListResponseIF extends SlackResponse {
+  @JsonProperty("channels") // just. wow.
+  List<Conversation> getConversations();
+
+  Optional<ResponseMetadata> getResponseMetadata();
```

Comment:
This is already part of every response in https://github.com/HubSpot/slack-client/blob/master/slack-base/src/main/java/com/hubspot/slack/client/models/response/SlackResponse.java#L7. No need to explicitly include it here.